content/en/examples/deployments/deployment-with-low-priorityclass.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: placeholder-deployment
```

```suggestion
  name: capacity-reservation
```
(consider changing the filename of the manifest too)